### ISSUE TITLE
Add nav buttons

### ISSUE THREAD

**Issue by nileshgulia1** [LinuxChixIndia/LinuxChixIndia.github.io#21]
It can be good if we add nav buttons to the website may be like https://www.linuxchix.org/. Need some discussion from members.

**Comment by CuriousLearner** [comment]
Right now it's just a single page website. Do you intend to include anchor links in nav panel instead?

**Comment by nileshgulia1** [comment]
Yes exactly,The idea is to add anchor links(```HOME```,```ABOUT```,```CONTACT``` or like so) which point to some external page specified by members.Can i work on it ? and also can you give me the links it will point to?:)

**Comment by CuriousLearner** [comment]
Yes, go ahead. There are no specific links, if there are no anchor tags defined right now, so you've to go and define them before including them in the navbar. I hope that makes sense?
